src/components/Uploader/index.js

```diff
@@ -32,7 +32,7 @@ const UploadError = ({error}) =>
   <p className={style.error}>{`${error.message}. ${error.instruction}.`}</p>
 
 const UploaderPure = ({method, side, onImageSelected, error, changeFlowTo, allowCrossDeviceFlow}) =>
-  <div className={classNames(style.uploaderWrapper, {[style.crossDeviceClient]: !allowCrossDeviceFlow})}>
+  <div className={classNames(style.uploaderWrapper, {[style.noCrossDevice]: !allowCrossDeviceFlow})}>
```

Comment:
I think `noCrossDevice` is a misleading name for this class. That class only gets added when you are on the cross device client. At the moment we use this prop `allowCrossDeviceFlow` to display the SwitchDevice component (the component that takes you to the cross device link), it returns `true` if it's desktop and it's not the cross device client, so maybe we should rename it to reflect that.